docs/build/develop-services.md

```diff
@@ -299,4 +299,157 @@ result: "Hello, fluence"
 3> 
 ```
 ## Mounted binaries
-To unlock some features not provided for wasm, exists a mounted binaries API - a way to call programs on host machine from wasm services.  TODO
\ No newline at end of file
+To unlock some options not possuble in pure wasm, exists a mounted binaries API - a way to call programs on host machine from wasm services. It is similar to the importing functions, the only differences are that `wasm_import_module` must be `"host"` and signatures are predefined.
```

Comment:
```suggestion
To unlock some options not possible in pure wasm, there is a mounted binaries API, a way to call programs on a host from Wasm services. It is very similar to functions importing, and the only difference is that `wasm_import_module` must be `"host"` and signatures are predefined.
```

Reply:
Perhaps, a more simple text is possible:
There exists a mounted binary API, a way to call programs on a host from Wasm services. It allows one to unlock some options not available in pure Wasm. The API is similar to function importing. The only difference is that `wasm_import_module` must be `"host"` and signatures are predefined.